.github/workflows/main.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Setup LXD
         uses: canonical/setup-lxd@main
         with:
-          channel: 5.17/stable
+          channel: 5.19/stable
```

Comment:
I think we should use `5.0/stable` here, as it is the LTS release. `5.19` support will be dropped when `5.22` is released.

Reply:
Default version used in `setup-lxd` is `5.0/stable`. removed the channel parameter.